core/src/main/scala/scalaz/EitherT.scala

```diff
@@ -215,6 +215,10 @@ final case class EitherT[F[_], A, B](run: F[A \/ B]) {
 }
 
 object EitherT extends EitherTInstances {
+  def F[F[_], A, B](a: F[A \/ B]): EitherT[F, A, B] = apply(a)
+  def A[F[_]: Applicative, A, B](a: A): EitherT[F, A, B] = apply(Applicative[F].point(-\/(a)))
+  def B[F[_]: Applicative, A, B](b: B): EitherT[F, A, B] = apply(Applicative[F].point(\/-(b)))
```

Comment:
`pureRight`? Or just `pure`?

Reply:
Or just `right` and rename the current `right` to `rightT`.

Reply:
I'd prefer a new name, e.g. `pureRight`, to redefining existing methods. But also, 9 characters for a constructor feels kinda long.